deepl-connector-product/pom.xml

```diff
@@ -26,6 +30,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.7</version>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <configuration>
+              <skip>${skip-readme}</skip>
+              <tasks>
+                <copy todir="${project.build.directory}">
+                  <fileset dir="../../" includes="doc/**" />
+                </copy>
+                <copy file="README.md" tofile="${project.build.directory}/README.md" />
+                <loadfile property="variables.yaml" srcFile="${variables.yaml.file}" encoding="UTF-8" />
+                <replace file="${project.build.directory}/README.md" token="@variables.yaml@" value="${variables.yaml}" />
```

Comment:
I think we should add this pattern replacement to the product-template: 
- almost every product needs configuration and therefore explain their variables ... now we always duplicate the variables for that purpose into the readme, which is prone to errors.
Do you agree @alexsuter

Reply:
Or maybe we should have a variables editor which shows which variables can be overridden.